Source/Project64/Multilanguage/Language Class.cpp

```diff
@@ -269,13 +270,16 @@ void CLanguage::LoadDefaultStrings (void)
 	DEF_STR(ROM_USE_TLB,         L"Use TLB");
 	DEF_STR(ROM_REG_CACHE,       L"Register caching");
 	DEF_STR(ROM_DELAY_SI,        L"Delay SI Interrupt");
-	DEF_STR(ROM_SP_HACK,         L"SP Hack");
+	DEF_STR(ROM_VIREFRESH,       L"Vi Refresh Rate: ");

```

Comment:
I didn't know that. I wasn't sure if Fast SP and SP Hack were related. Some of the strings are never used, so they may have changed in one file, but never updated in another.